packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/brick.yaml

```diff
@@ -14,3 +14,4 @@ vars:
   - enable_feature_deeplinks
   - enable_feature_widget_toolkit
   - enable_social_logins
+  - enable_internationalisation
```

Comment:
Same as above

Reply:
this var was changed to `enable_change_language`